### ISSUE TITLE
Running Error With Knowledge Distillation 

### ISSUE THREAD

**Issue by jiangguisyp** [keras-team/keras-io#269]
It occurs that "compile() got an unexpected keyword argument 'loss'" error when running Knowledge Distillation example with tf1.14 and tf1.15. Anyone can help?

**Comment by graghavan007** [comment]
i get the same error using tf2.0. Any answers please ?

**Comment by meghakalia** [comment]
I am receiving the same error when running the example. I am using tf2.0.0

**Comment by rohithmars** [comment]
hello all,
can someone address this issue? i am also using tf 2.0.0. 

The code is actually well written and easy to follow compared to other KD codes. Please address it. Thanks

**Comment by Kennethborup** [comment]
Hi all,

Unfortunately, it appears that the errors you get with TF2.0.0 (and below) are due to the way the model (`Distiller`) is compiled internally, and I am in no way comfortable enough in these internals to help with this error.
Though, it appears to work in the Colab with TF2.2.0 and above.

@rohithmars: I am glad that you find the example well written and clear! 

@jiangguisyp: Given Keras was moved into TF at TF2, I would not expect the example to work with TF1.x as you would need the stand-alone Keras and likely other refactoring of the code as well.

**Comment by sachinprasadhs** [comment]
Hi,

Thanks for reporting the issue.

Since we no longer support Tensorflow 1.x issues, please consider using the latest Tensorflow version 2.13 and follow the below tutorial which has no known issues.
https://keras.io/examples/vision/knowledge_distillation/
